config/zookeeper_persister.go

```diff
@@ -246,7 +258,9 @@ func (z *ZkConfigPersister) currentConfigEventListener() (<-chan zk.Event, error
 	defer z.Unlock()
 
 	z.configs = configs
-	z.config = string(config)
+	z.config = latestHash
```

Comment:
is it expected that `latestHash` might be an empty string (if `currentVersion < latestHashVersion`)

Reply:
nvm, that's impossible